src/Resolvers/Microsoft.DotNet.NativeWrapper/Microsoft.DotNet.NativeWrapper.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(ResolverTargetFramework);net472</TargetFrameworks>
+     <IsAotCompatible Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">true</IsAotCompatible>
```

Comment:
For my understanding, why did we use net8.0 here?

Reply:
When I didn't have the condition it gave me an error saying the property wasn't supported for .NET Framework, and the error suggested this condition.  I think the property is maybe supported on .NET 6 and higher.